src/rules/exports-last.js

```diff
@@ -0,0 +1,35 @@
+function isExportStatement({ type }) {
+  // ES Module export statements
+  if (type === 'ExportDefaultDeclaration' || type === 'ExportNamedDeclaration') {
```

Comment:
Simpler:
```js
function isExportStatement({ type }) {
  return type === 'ExportDefaultDeclaration' || type === 'ExportNamedDeclaration';
}
```

And you should probably also test for ExportAllDeclaration (and add tests for that case)